managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -465,7 +773,7 @@ public void operationFailed(MetaStoreException e) {
 
                 LedgerInfo info = LedgerInfo.newBuilder().setLedgerId(lh.getId()).setTimestamp(0).build();
                 ledgers.put(lh.getId(), info);
-
+                initializeStreamingOffloader();
```

Comment:
The streaming offload should be started after the newest ledger is created, at ManagedLedger initialization the newest ledger may not be created yet because related works are asynchronous.